spec-tests/src/test/java/org/candlepin/spec/consumers/ConsumerCheckinSpecTest.java

```diff
@@ -107,22 +106,6 @@ public void shouldUpdateLastCheckinTimeWhenFetchingEntitlementCertSerials() thro
             .doesNotReturn(initialCheckin, ConsumerDTO::getLastCheckin);
     }
 
-    @Test
-    public void shouldUpdateLastCheckinTimeOnHypervisorCheckin() throws Exception {
-        ApiClient adminClient = ApiClients.admin();
-        OwnerDTO owner = adminClient.owners().createOwner(Owners.random());
-        ConsumerDTO consumer = adminClient.consumers().createConsumer(Consumers.random(owner));
-        ApiClient consumerClient = ApiClients.ssl(consumer);
-        OffsetDateTime initialCheckin = consumer.getLastCheckin();
-
-        consumerClient.hypervisors().hypervisorUpdate(owner.getKey(), new HashMap<>(), false);
```

Comment:
Instead of fully removing the test, would it be valid to just replace the `hypervisorUpdate` method with the `hypervisorUpdateAsync` method?

Reply:
The next method in the class is the async one so this can be cut.